### PR TITLE
Ensure ECU-TEST 2023.3 Compatibility

### LINKED ISSUE #71: Ensure ECU-TEST 2023.3 Compatibility
### Describe the feature request

As a TraceTronic member we want to ensure compatibility to ECU-TEST 2023.3.

### Upstream changes

* [x] update compatibility hint

### Acceptance criteria

* [x] support ECU-TEST 2023.3

### PR DESCRIPTION
**Pull request by MartinGroscheTT**
Resolves #71

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,24 +119,25 @@ If the problem still exists search the following list of issues for possible sol
 
  <details>
     <summary><a href="https://www.tracetronic.com/products/ecu-test">ECU-TEST</a>/
-    <a href="https://www.tracetronic.com/products/trace-check/">TRACE-CHECK</a> 2021.4.1 or higher </summary>
+    <a href="https://www.tracetronic.com/products/trace-check/">TRACE-CHECK</a> 2022.4 or higher </summary>
```
Comment (tt-marcel) [inline README.md:122]: The current TRACE-CHECK version is 2023.3, the ticket only asks for ECU-TEST to be updated, but I think compatibility with TRACE-CHECK should also be checked. => `2023.3`

There are many TRACE-CHECK versions between these versions. Are they checked too?

File: README.md [hunk 2]
```diff
@@ -119,24 +119,25 @@ If the problem still exists search the following list of issues for possible sol
...
 
 </details>
 <details>
     <summary>optional: <a href="https://www.tracetronic.com/products/test-guide">TEST-GUIDE</a> 1.114.3 or higher</summary>
 
-| Version |         3.0         |
-|:-------:|:-------------------:|
-| 1.147.0 | :heavy_check_mark:  |
-| 1.142.0 | :heavy_check_mark:  |
+| Version |    latest - 3.0    |
```
Comment (tt-marcel) [inline README.md:137]: It seems we forgot to add the [3.1 release](https://github.com/jenkinsci/ecu-test-execution-plugin/releases/tag/ecu-test-execution-3.1) to this table. Please add the column for 3.1

Comment (MartinGroscheTT) [reply README.md:137]: latest includes version 3.1

**Review by tt-marcel — COMMENTED** [review]
All review comments are resolved.

**Review by tt-marcel — APPROVED** [review]
All review comments are resolved ok.

**Pull request merged**